.travis.yml

```diff
@@ -1,35 +1,33 @@
 language: php
 
-php:
-# - 5.3 # requires old distro, see below
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7.0
-  - 7.1
-  - 7.2
-  - hhvm # ignore errors, see below
-
```

Comment:
Nit: Unrelated change, but I don't want to hold this back because of this :+1:

Reply:
Dropped the `-VER`

Reply:
I meant to refer to the whole `php:` vs `matrix:` block, but it's alright as-is IMHO :+1: